appIcons.js

```diff
@@ -623,6 +783,12 @@ const MyAppIcon = new Lang.Class({
         let spacing = radius + borderWidth; // separation between the dots
         let n = this._nWindows;
 
+        // Lightly adjust the styling in case of unity7 backlit mode
+        if (this._dtdSettings.get_boolean('unity-backlit-items') === true) {
```

Comment:
Maybe here we can check if `custom-theme-customize-running-dots` is true, and not override the padding and width from the settings. So something like:
```
if (this._dtdSettings.get_boolean('unity-backlit-items') &&
    !this._dtdSettings.get_boolean('custom-theme-customize-running-dots')) {
        padding = 1.45;
        borderWidth = 2;
}
```